index.html

```diff
@@ -6,30 +6,68 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Awesome Books</title>
     <link rel="stylesheet" href="index.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
 </head>
 <body>
-    <section class="all_books">
-        <h1 class="page-title">All awsome books</h1>
 
+    <nav class="navbar navbar-dark navbar-expand-lg bg-dark fixed-top">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">Awesome Books</a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#list-menu" aria-controls="list-menu" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
- [ ] **Required:**Great job guys! However, you are handling it the wrong way. You have attended to display a section using target ids but the goal of this project is to attempt to build a SPA using vanilla JS.

So only the active button's content would display while other sections are hidden completely from the page.

Reply:
Thanks for the advice, I am going to change it to follow the SPA rules